src/Stratis.Bitcoin.IntegrationTests/SegWitTests.cs

```diff
@@ -14,9 +14,9 @@ public class SegWitTests
         [Fact]
         public void TestSegwit_MinedOnCore_ActivatedOn_StratisNode()
         {
-            using (NodeBuilder builder = NodeBuilder.Create(version: "0.15.1"))
+            using (NodeBuilder builder = NodeBuilder.Create(this))
             {
-                CoreNode coreNode = builder.CreateBitcoinCoreNode();
+                CoreNode coreNode = builder.CreateBitcoinCoreNode(version: "0.15.1");
```

Comment:
This appears to be the only test that uses the bitcoind? Should default version to "0.15.1" so that it's easy to updated all instances to a new version. Not really a problem right now as there is only this test, but maybe in future we will have more.